exercises/accumulate/AccumulateTests.elm

```diff
@@ -12,18 +15,21 @@ square x =
 
 tests : Test
 tests =
-    suite "Accumulate"
+    describe "Accumulate"
         [ test "[]] Accumulate"
-            (assertEqual [] (accumulate square []))
+            (\() -> Expect.equal [] (accumulate square []))
```

Comment:
I think we should do it, but perhaps it's something we can leave for a later pull request, or just do incrementally.